CHANGELOG.md

```diff
@@ -5,8 +5,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-===================
+## [v23.0.0] - 2023-05-15
+
+### Notes
+
+- Testing is done against Python >=3.11. Minimum requirement is now Python 3.5
+- The repository has been refactored to use `pyproject.toml` as the source of truth for dependency requirements
+- Added `.python-version` file to keep track of supported Python version
+
+### Breaking
+- accept grpc endpoint or /graphql endpoint in from_cloud function (#190)
```

Comment:
We also remove  the slash function, we should mention that too. Looks good otherwise.